servicetest/capabilities/test_caps.py

```diff
@@ -230,3 +230,17 @@ def test_defaults_are_applied(self):
             assert my_env_var == "TEST_ENV_VALUE"
         finally:
             sc.terminate()
+
+    def test_set_same_cap_twice(self):
+        """ Test that setting the same capability doesn't result in crashes
```

Comment:
Maybe we can be even a little more clear here, and describe that we are expecting a D-Bus exception of a certain kind and not a crash which results in a timeout error on this end (the client).